src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -68,6 +70,15 @@ export const JiraConnectEnterprisePost = async (
 		}
 
 		req.log.debug(`No existing GitHub apps found for url: ${gheServerURL}. Making request to provided url.`);
+
+		// Setting time out check
+		res.setTimeout(TIMEOUT_PERIOD, () => {
```

Comment:
Would it make sense to also use axios' timeout? 
<img width="330" alt="image" src="https://user-images.githubusercontent.com/20631664/186311282-a442b566-867a-4a91-8402-54ae5f0b446b.png">

Reply:
Replaced this by using axios's timeout...